code/src/App.js

```diff
@@ -1,9 +1,13 @@
-import React from 'react'
+import React from "react";
+
+import Form from "./components/Form";
 
 export const App = () => {
   return (
-    <div>
-      Find me in src/app.js!
-    </div>
-  )
-}
+    <section className="app">
+      <div className="main-container">
```

Comment:
From what I was able to notice, for the "app" you have a margin set in css and for the "main-container" you're setting the layout to display: flex and justify-content: center. I think you could combine this styling into one of these elements and then you could remove the other one and make the code a little bit cleaner that way :)

Reply:
Thank you I made this change! :)